application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -87,22 +87,15 @@
     #set ($checked = '')
     #if ($isChecked)
       #set ($checked = 'checked')
-      #set ($nbvotes = $voteCount.get($proposal))
-      #set ($nbvotes = $nbvotes + 1)
-      #set ($discard = $voteCount.put($proposal, $nbvotes))
     #end
     &lt;td&gt;
       #if ($xcontext.action == 'export')
         #if($isChecked)X#end
       #else
         &lt;label&gt;
-          #if ($isRadio)
-            &lt;input type="radio" #if($isCurrentUser)id="${user}_$proposal.hashCode()"#end name="$user"
-              value="$proposal.hashCode()" $checked $disabled /&gt;
-          #else
-            &lt;input type="checkbox" #if($isCurrentUser)id="$proposal.hashCode()"#end name="$proposal.hashCode()"
-              $checked $disabled /&gt;
-          #end
+            &lt;input type="#if($isRadio)radio#{else}checkbox#end"
+    #if($isCurrentUser)id="xpoll-proposal$proposal.hashCode()"#end name="$user" value="$proposal" $checked
+    $disabled /&gt;
```

Comment:
Strange code formatting. The first line should have 2 space indentation and the following ("wrapped") lines should have an extra 2 space indentation. Check the code you removed.